src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,18 @@
-// Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
-);
+import { User } from '../../types/User';
+
+type Props = {
+  person: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ person }) => {
+  const {
+    name,
+    email,
+  } = person;
```

Comment:
that's ok to one line it
```suggestion
  const { name, email } = person;
```